client/src/constants/messages.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
+
 /**
- * Standard welcome message texts
- *
- * Notes:
+ * Standard Welcome Message Texts
```

Comment:
Are the items distinguished here "welcome" messages?

Reply:
Based on precedence, yes.

<details>
<summary>Details</summary>

_The word "welcome" matches the adjectives used to describe this text since [initial message](https://github.com/TACC/Core-Portal/blob/cc8e425/client/src/constants/welcomeMessages.js#L8-L9) (originally from [FP-376](https://jira.tacc.utexas.edu/browse/FP-376), originally named "Dashboard Welcome Message") to [implementation](https://github.com/TACC/Core-Portal/blob/dc2af86/client/src/components/_common/Section/SectionMessages.js#L51-L60) (via https://github.com/TACC/Core-Portal/pull/347)._

</details>

Based on semantics, no.

<details>
<summary>Details</summary>

_These can be described as usage hints or usage guidance messages, or introductory messages. Actually, "introduction" and "introductory" are the adjectives used since [the meeting early last year](https://confluence.tacc.utexas.edu/pages/viewpage.action?pageId=123928721#UI-Requests-2020-03-11,FronteraUI/UXCoordination-IntroduceApplication) that resulted in this "temporary" solution._

</details>

If this is a concern, then I have ideas for solutions toward a more semantic description.

<details>
<summary>Ideas</summary>

- New ticket to rename `/Welcome/i` to `/Intro/i`.
- In this ticket, describe the use of the messages as for section introduction (maybe also link to this discussion so the message can be succinct but still provide details).

</details>

Thoughts?

Reply:
Agree, `Intro` does make sense, let's go ahead with that idea and make the update part of this PR.

Reply:
Okay. I have created a ticket, [FP-1165](https://jira.tacc.utexas.edu/browse/FP-1165).

<details>
<summary>Regarding performing FP-1165 within this ticket…</summary>

> Renaming and testing that rename within this PR would exceed the scope of this ticket as I estimated for it in this sprint given my limited bandwidth.
> 
> <details>
> <summary>intended scope of (noop) ticket</summary>
> 
> - distinguish constants
> - comment changes
> - whitespace changes
> - discussion
> - no testing
> 
> </details>

</details>

Reply:
I just meant that we should change the text in the comment from `Standard Welcome Message..` to `Standard Intro Message..`

Reply:
[Updated.](https://github.com/TACC/Core-Portal/pull/445/commits/81ce36aea6ef5e781ed090245ebccd63b7a77d0f)